Backend/src/main/java/com/example/Backend/config/UserConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.example.Backend.config;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+@Configuration
+public class UserConfiguration {
+    @Bean
+    @Primary
+    public OpenAPI userOpenAPI() {
+        return new OpenAPI()
+                .info(new Info().title("Login and Registration API")
+                        .description("Endpoint for user login and Registration"));
+    }
+
```

Comment:
add Bean for Springboot validator interface which could help us validate our DTO and entity. Refer to section implement Validator interface  in https://phoenixnap.com/kb/spring-boot-validation-for-rest-services